Makefile

```diff
@@ -22,8 +22,12 @@ endif
 SAIL_CHECK_SRCS = riscv_addr_checks_common.sail riscv_addr_checks.sail riscv_misa_ext.sail
 SAIL_DEFAULT_INST = riscv_insts_base.sail riscv_insts_aext.sail riscv_insts_cext.sail riscv_insts_mext.sail riscv_insts_zicsr.sail riscv_insts_next.sail riscv_insts_hints.sail
 SAIL_DEFAULT_INST += riscv_insts_fext.sail riscv_insts_cfext.sail
+SAIL_DEFAULT_INST += riscv_insts_crypto.sail
```

Comment:
Hm, thinking this through a bit, this probably should be named riscv_insts_zcrypto.sail or whatever your ISA string is. Do you have one proposed yet?

Reply:
Thats a good point. Yes, we are the `K` extension. I'll change "crypto" in the file names to `kext` in keeping with the others.